README.md

```diff
@@ -1,3 +1,71 @@
-# Go here -> https://github.com/gnolang/gno/issues/390
+# Game of Realms
 
-_note: This repo will eventually be used more actively, but right now we prefer centralizing the knowledge on the main repo. Please, do not open issues here._
+[Go Here!](https://github.com/gnolang/gno/issues/390)
+
+The worldwide competition to find the best contributors to Gno.land is currently in phase one – for advanced developers only.
+
+Once the necessary tools to start phase two are ready, we’ll open up the competition to newer devs and non-technical contributors.
+
+If you want to stack ATOM rewards and play a key role in the success of Gno.land and web3, read more about Game of Realms or open a [PR](https://github.com/gnolang/gno/) today.
+
+⚠️ We ask that you tag your PR with the GoR label!
+
+## Phase I. (ongoing)
+
+<details>
+<summary>Evaluation DAO (30%)</summary>
+
+### Evaluation DAO
+
+This complex challenge seeks your skills in DAO development and implementation and is one of the most important challenges of phase one. The Evaluation DAO will ensure that contributions in Game of Realms and the Gno.land platform are fairly rewarded.
+
+- [x] Clarifying this issue — [100% completed]
```

Comment:
shall we add the links to the issues to have more context? If the issue contains its own checklist, the process will be updated automatically.

Reply:
```suggestion
- [x] https://github.com/gnolang/gno/issues/407
```